### PR TITLE
extend rtf conf.py tmpl

### PR DESCRIPTION
**Pull request by paoloromolini**
Extend doc_builder/conf.py.tmpl to add documents_settings.yml data

### CODE REVIEW

**Review by xrmx — CHANGES_REQUESTED** [review]
You lost a doc_builder in the template path

File: readthedocs/docsitalia/templates/conf.py.tmpl [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    },
+    {% endfor %}
+]
+try:
+    with open('document_settings.yml') as rtd:
+        data = rtd.read()
+        data = yaml.safe_load(data)
+except:
+    data = {}
+context['extra_data'] = data
```
Comment (xrmx) [inline readthedocs/docsitalia/templates/conf.py.tmpl]: I'd call this docsitalia instead of extra_data

Comment (xrmx) [reply readthedocs/docsitalia/templates/conf.py.tmpl]: or docsitalia_data

**Review by xrmx — DISMISSED** [review]

File: readthedocs/settings/base.py [hunk 2]
```diff
@@ -86,6 +86,9 @@ def INSTALLED_APPS(self):  # noqa
             'messages_extends',
             'tastypie',
 
+            # docsitalia
```
Comment (xrmx) [inline readthedocs/settings/base.py]: move this back please

**Review by xrmx — CHANGES_REQUESTED** [review]
please squash so we can merge, thanks!

**Review by yakky — APPROVED** [review]

**Review by xrmx — APPROVED** [review]

**Pull request merged**